src/test/groovy/com/stormpath/tck/forgot/ChangePasswordIT.groovy

```diff
@@ -168,12 +176,18 @@ class ChangePasswordIT extends AbstractIT {
     @Test(groups=["v100", "html"])
     public void changeRedirectsToErrorUriForInvalidSptokenWhenPosting() throws Exception {
 
-        given()
+        saveCSRFAndCookies(ForgotRoute)
+
+        def req = given()
             .accept(ContentType.HTML)
             .contentType(ContentType.URLENC)
             .queryParam("sptoken", "NOTEVENCLOSETOVALID")
             .param("password", "foobar123!")
-        .when()
+            .param("confirmPassword","foobar123!")
```

Comment:
This should not be in the tests; password confirmation should be disabled by default:

https://github.com/stormpath/stormpath-framework-spec/blob/20a7dfc52a95ee45969fae59aaa3f3bd161659db/example-config.yaml#L101